packages/node-resolve/README.md

```diff
@@ -221,7 +214,7 @@ export default ({
 })
 ```
 
-## Resolving Require Statements
+## Resolving require statements
```

Comment:
please try not to reformat. this change needs to be reverted please

Reply:
Little confused how that happened! Definitely wasn't deliberate.